docs/eventlisteners.md

```diff
@@ -297,94 +306,58 @@ spec:
                   cpu: "500m"
 ```
 
-With the help of `CustomResource` user can specify any dynamic object which adheres to the Contract described below.
-
-##### Contract
-For Knative or any new CRD should satisfy [WithPod{}](https://github.com/knative/pkg/blob/master/apis/duck/v1/podspec_types.go#L41)
-
-**Spec**
-```spec
-   spec:
-     template:
-       metadata:
-       spec:
-```
-**Status**
-```status
-type EventListenerStatus struct {
-  duckv1beta1.Status `json:",inline"`
-
-  // EventListener is Addressable. It currently exposes the service DNS
-  // address of the the EventListener sink
-  duckv1alpha1.AddressStatus `json:",inline"`
-}
-```
-
-##### Note
-For both `CustomResource` and `kubernetesResource` the allowed values for `PodSpec` and `Containers` are
-```text
-ServiceAccountName
-NodeSelector
-Tolerations
-Volumes
-Containers
-- Resources
-- VolumeMounts
-- Env
-```
+## Specifying target namespaces
```

Comment:
Do you think its clearer if we say something like Specifying Triggers in target namespaces? (And for the label selector below, - Specifying Triggers using labels?)

Reply:
Changed both to "Constraining `EventListeners` to specific namespaces/labels" let me know if that works.